docker/rust_base/Dockerfile

```diff
@@ -1,14 +1,20 @@
 FROM fleupold/dfusion-solver-base:v2
 
 RUN apt-get update \ 
-    && apt-get install -y --no-install-recommends curl ssh git libc-dev gcc openssl pkg-config libssl-dev \
+    && apt-get install -y --no-install-recommends curl ssh git libc-dev gcc openssl pkg-config libssl-dev libpq-dev \
```

Comment:
This is a dependency for communicating with PSQL?

Do we think it would make sense to put these dependencies on separate lines so we can include a brief comment about their necessity? Or is this too much more than necessary?

Reply:
I agree with your point/concern.

Initially I installed the packages further below when I also create the empty packet structure. However this required to do another command which is not great for the docker image size.

Maybe we can land on a middle ground where I put them in a separate line in the same command and add a comment about that they are only for the graph listener.

Reply:
Works for me!